ui/src/app/shared/header/header.component.ts

```diff
@@ -245,7 +261,7 @@ export class HeaderComponent implements OnInit, AfterViewInit, AfterViewChecked,
       j => this.onHoldStatuses.includes(j.status)).length;
   }
 
-  private refreshChips(query: string): void {
+  refreshChips(query: string): void {
```

Comment:
nit: why the change?

Reply:
Happened during debugging the tests; fixed.